src/group_theory/nilpotent.lean

```diff
@@ -181,6 +181,15 @@ lemma upper_central_series_is_ascending_central_series :
   is_ascending_central_series (upper_central_series G) :=
 ⟨rfl, λ x n h, h⟩
 
+lemma upper_central_series_mono (n : ℕ)
+: upper_central_series G n ≤ upper_central_series G n.succ :=
```

Comment:
style - `:` goes on the end of a line:
```suggestion
lemma upper_central_series_mono (n : ℕ) :
    upper_central_series G n ≤ upper_central_series G n.succ :=
```
but why not state it like you mean it?
```suggestion
lemma upper_central_series_mono : monotone (upper_central_series G) :=
monotone_nat_of_le_succ $ λ n,
```